### ISSUE TITLE
I (naïvely) expected `dtt/mset!` to broadcast for me

### ISSUE THREAD

**Issue by harold** [cnuernber/dtype-next#53]
With an image:

```clojure
> (def i (image/new-image 4 4 :byte-abgr))
#'i
```

`dtt/mset!` on a scalar fills the image:
```clojure
> (dtt/mset! (dtt/ensure-tensor i) 255)
#<tech.v3.tensor_api.DirectTensor@8c1d245 #tech.v3.tensor<uint8>[4 4 4]
[[[255 255 255 255]
  [255 255 255 255]
  [255 255 255 255]
  [255 255 255 255]]
 [[255 255 255 255]
  [255 255 255 255]
  [255 255 255 255]
  [255 255 255 255]]
 [[255 255 255 255]
  [255 255 255 255]
  [255 255 255 255]
  [255 255 255 255]]
 [[255 255 255 255]
  [255 255 255 255]
  [255 255 255 255]
  [255 255 255 255]]]>
```

I was hoping to clear w/ a color other than white by doing this:
```clojure
> (dtt/mset! (dtt/ensure-tensor i) [0x20 0x20 0x20 0xFF])
Execution error at tech.v3.tensor.tensor-copy/bit-blit! (tensor_copy.clj:78).
Src shape ([4 4]) differs from dst shape ([4 4 4])
```

That didn't work, so I ended up doing this:
```clojure
> (-> [0x20 0x20 0x20 0xFF]
      (dtt/broadcast [4 4])
      (dtt/broadcast [4 4 4])
      (dtt/nd-copy! (dtt/ensure-tensor i)))
#<tech.v3.tensor_api.DirectTensor@5d85d957 #tech.v3.tensor<uint8>[4 4 4]
[[[32 32 32 255]
  [32 32 32 255]
  [32 32 32 255]
  [32 32 32 255]]
 [[32 32 32 255]
  [32 32 32 255]
  [32 32 32 255]
  [32 32 32 255]]
 [[32 32 32 255]
  [32 32 32 255]
  [32 32 32 255]
  [32 32 32 255]]
 [[32 32 32 255]
  [32 32 32 255]
  [32 32 32 255]
  [32 32 32 255]]]>
```

Which did work, but was more lines.

Neither urgent nor important, but interested to hear your thoughts.

https://user-images.githubusercontent.com/7443/150688662-b551e763-e013-4b1a-9f27-b99429a09248.mp4

**Comment by cnuernber** [comment]
I think it should braodcast.

**Comment by cnuernber** [comment]
Fixed in 8.064
